infra/nnfw/python/nnfwapi/libnnfw_api_pybind.py

```diff
@@ -1,3 +1,4 @@
+import numpy as np
```

Comment:
As I wrote in previous PR, I don't understand why `infra` needs to have python code, which seemingly be for user, not for maintainer.

Reply:
Recently, we said about this problem but we didn't update this code. I'm sorry.
We'll place this file not `infra` but `runtime/onert/python`.